guides/common/modules/proc_providing-feedback-on-red-hat-documentation.adoc

```diff
@@ -0,0 +1,24 @@
+:_content-type: PROCEDURE
+
+[preface]
```

Comment:
```suggestion
```

Reply:
@maximiliankolb , removed the first line. The [preface] must stay there because of the structure of downstream docs. Thanks for understanding!